portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,23 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <span>
```

Comment:
[<span>](https://www.w3schools.com/tags/tag_span.asp) is an "inline container" and as such shouldn't have block containers like `<div>`.

What you can do instead is replace `<span>` with `<div>` and give it an `id` and style using the ID in the CSS.